### DISCUSSION TITLE
[Feature Request] Notification

### DISCUSSION THREAD

**Discussion by kdipto1** [heroui-inc/heroui#1966] [Feature requests]
### Is your feature request related to a problem? Please describe.

A notification is a formal notice, request, or warning. 

### Describe the solution you'd like

A notification is a formal notice, request, or warning. 

### Describe alternatives you've considered

A notification is a formal notice, request, or warning. 

### Screenshots or Videos

![Screenshot from 2023-11-11 01-16-56](https://github.com/nextui-org/nextui/assets/96812332/3a04e0ec-8d72-492d-ade3-2a65129c2904)

**Comment by yWilliamLima** [comment]
up

**Comment by ArianSha** [comment]
There is already a feature request for toasts:
https://github.com/nextui-org/nextui/discussions/314

**Comment by wingkwong** [comment]
see toast: #314
